app/models/page.rb

```diff
@@ -48,5 +62,11 @@ def switch_plugins
       PagePluginSwitcher.new(self).switch(liquid_layout)
     end
   end
+
+  def primary_image_is_owned
```

Comment:
I wrote this yesterday. The idea is that the page has many images, but that only one is the "primary image" for the cover photo. this validates that the primary image is one of the images from the has_many relationship.